src/components/pages/StorageEditPage.tsx

```diff
@@ -32,7 +32,7 @@ export default ({ storage }: StorageEditProps) => {
 
   const removeCallback = useCallback(() => {
     messageBox({
-      title: `Remove "${storage.name}" storage`,
+      title: `Delete "${storage.name}" storage`,
```

Comment:
If I do that, I don't know what will happen to other users that aren't using the app in English since I can only provide the text in in that language. I believe as for the other languages, it might show up as blank. Although I can use _Google Translate_ if grammar isn't much of an issue.

Reply:
Sorry, but I do not know any method in concatenating the string translations w/ template literals such as above.

Reply:
@jhdcruz you can use this feature of `i18next` to pass variable to the translate string (look at the `count` variable in translate string):
https://www.i18next.com/translation-function/context#combining-with-plurals

Reply:
Resolved in https://github.com/BoostIO/BoostNote.next/pull/446/commits/0bdcd12f8c839113329e962a9859878ee696d2d6